utils/lint_test_cases.sh

```diff
@@ -4,7 +4,19 @@ OUT=0
 SCT_SCYLLA_VERSION=4.4.1 ./sct.py lint-yamls -i '.yaml' -e 'multi-dc,multiDC,multidc,multiple-dc,rolling,docker,artifacts,private-repo,ics/long,scylla-operator,gce,jepsen,repair-based-operations'
 OUT=$(($OUT + $?))
 
-SCT_GCE_IMAGE_DB=image SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b gce -i 'multi-dc,multiDC,multidc,multiple-dc,rolling,artifacts,private-repo,gce,jepsen' -e 'docker,azure'
+SCT_GCE_IMAGE_DB=image SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b gce -i 'rolling,artifacts,private-repo,gce,jepsen' -e 'multi-dc,docker,azure'
+OUT=$(($OUT + $?))
+
+echo "2 region multi dc yamls"
+SCT_GCE_IMAGE_DB=image SCT_GCE_DATACENTER="us-east1 us-west1" SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b gce -i 'multi-dc-rolling,3h-multidc,multidc-4h,topology-changes,snitch-multi-dc,manager-regression-multiDC' -e 'docker,azure,shutdown'
+OUT=$(($OUT + $?))
+
+echo "3 region multi dc yamls"
+SCT_GCE_IMAGE_DB=image SCT_GCE_DATACENTER="us-east1 us-west1 eu-north1" SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b gce -i '24h-multidc,large-cluster,counters-multidc' -e 'docker,azure,shutdown'
+OUT=$(($OUT + $?))
+
+# Files that are named "multiple-dc" but are single dc
+SCT_GCE_IMAGE_DB=image SCT_GCE_DATACENTER=us-east1 SCT_SCYLLA_REPO='http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2021.1.repo' ./sct.py lint-yamls -b gce -i 'shutdown'
```

Comment:
this one file, please state while one it is, and doesn't seem to be used anywhere, sound like it's need to be remove.

I'm o.k. with not covering it at all.